website/docs/d/is_security_groups.html.markdown

```diff
@@ -0,0 +1,115 @@
+		Nested scheme for **remote**:
+			- `address` - (Optional, String) The IP address.This property may add support for IPv6 addresses in the future. When processing a value in this property, verify that the address is in an expected format. If it is not, log an error. Optionally halt processing and surface the error, or bypass the resource on which the unexpected IP address format was encountered.
+			- `cidr_block` - (Optional, String) The CIDR block. This property may add support for IPv6 CIDR blocks in the future. When processing a value in this property, verify that the CIDR block is in an expected format. If it is not, log an error. Optionally halt processing and surface the error, or bypass the resource on which the unexpected CIDR block format was encountered.
+			- `crn` - (Optional, String) The security group's CRN.
+			- `deleted` - (Optional, List) If present, this property indicates the referenced resource has been deleted and providessome supplementary information.
+			Nested scheme for **deleted**:
+				- `more_info` - (Required, String) Link to documentation about deleted resources.
+			- `href` - (Optional, String) The security group's canonical URL.
+			- `id` - (Optional, String) The unique identifier for this security group.
+			- `name` - (Optional, String) The user-defined name for this security group. Names must be unique within the VPC the security group resides in.
+		- `type` - (Optional, Integer) The ICMP traffic type to allow.
+	- `targets` - (Required, List) The targets for this security group.
+	Nested scheme for **targets**:
+		- `crn` - (Optional, String) The load balancer's CRN.
+		- `deleted` - (Optional, List) If present, this property indicates the referenced resource has been deleted and providessome supplementary information.
+		Nested scheme for **deleted**:
+			- `more_info` - (Required, String) Link to documentation about deleted resources.
+		- `href` - (Optional, String) The URL for this network interface.
+		- `id` - (Optional, String) The unique identifier for this network interface.
+		- `name` - (Optional, String) The user-defined name for this network interface.
+		- `resource_type` - (Optional, String) The resource type.
+	- `vpc` - (Required, List) The VPC this security group is a part of.
+	Nested scheme for **vpc**:
+		- `crn` - (Required, String) The CRN for this VPC.
+		- `deleted` - (Optional, List) If present, this property indicates the referenced resource has been deleted and providessome supplementary information.
+		Nested scheme for **deleted**:
+			- `more_info` - (Required, String) Link to documentation about deleted resources.
+		- `href` - (Required, String) The URL for this VPC.
+		- `id` - (Required, String) The unique identifier for this VPC.
+		- `name` - (Required, String) The unique user-defined name for this VPC.
```

Comment:
All required, optional needs to be removed here

Reply:
done.

Reply:
done.